frog/imports/ui/Repository.jsx

```diff
@@ -1,9 +1,230 @@
-import React, { Component } from 'react';
- 
-const Repository = () => { 
-  return(
-      <p>You are on the Repository app</p>
-  )
+import React, { Component, PropTypes } from 'react';
+import { Activities } from '../api/activities.js';
+import { createContainer } from 'meteor/react-meteor-data';
+import ReactDOM from 'react-dom';
+
+export default class Repository extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      nameFilterText:"",
+      //The elements in the following lists are the attributes of the activities
+      //we want to filter
+      plane: [],
+      type: [],
```

Comment:
It would have felt more natural to me to have `plane` and `type` be something like:
``` 
{
    Video: True/False,
    Quizz: True/False,
    Lecture: True/False,
}
{
    1: True/False,
    2: True/False,
    3: True/False
}
```